docs/data_structures.md

```diff
@@ -0,0 +1,74 @@
+For built area projects, each group has a height of three tasks and a width of approximately 40 tasks.
+For building footprint tasks, the group will contain about 100 footprints.
+
+| Parameter | Description |
+| --- | --- |
+| **Id** | Each group can be identified by its Id. |
+| **Tasks** | Each group contains several tasks. The information for all tasks of the group will be stored in an array. |
+| **Geometry** | The group geometry is defined by the union of all assigned task geometries. |
+| **Completed Count** |	Once a group has been completely mapped by a volunteer the completed count of the corresponding group will be raised by one. The completed count of the group is used to assess the overall progress of each project. For doing so the completed count is compared to the redundancy required (see Table 2). During the mapping process groups will be served in ascending completed count order. Thus, groups with low completed count will be served first. |
+
+## Tasks
+To create a new mapping task, the overall project extent is split up into many single tasks.
+Tasks are the smallest unit in the MapSwipe data model.
+
+### For built area projects
+They are derived from the area of interest by gridding it into many small equal-sized rectangular polygons.
+Each task corresponds to a specific tile coordinate from a tile map service (TMS) using a web Mercator projection as its geographical reference system.
+Therefore, each task is characterized by a geometry and its tile coordinates, which describe its x, y and z position.
+For the projects analysed in this project, the tiles for all tasks are generated at zoom level 18.
+Taking the latitude of each task location into account the satellite imagery has a maximum spatial resolution of ~ 0.6 meter at the equator.
+
+### For building footprint validation projects
+
+A task is the validation of the footprint of a single building. This would be a single "way" in OSM terminology. With the data used in input, multipolygons shouldn't appear, and if they do, they will be ignored here. Similarly, a single node will be ignored.
+The data required to complete a task includes:
+- the building geometry: 
+
+| Parameter | Description | Example: Built areas | Example: Building footprint |
+| --- | --- | --- | --- |
+| **Id** | Each task can be identified by its Id. The Id is a composition of its position in the corresponding tile map system, which can be described by the x, y and z coordinates. | "18-160175-127032" |  |
+| **Project Id** | The Id of the project this task relates to | "1" |  |
+| **Tile Z** | The z coordinate of the tile defines the zoom level. Greater values for z will correspond to higher spatial resolution of the corresponding image. For most regions Bing provides images up to zoom level 18. For aerial imagery or images captured by UAVs even higher z values are valid. | "18" | Similar to existing projects: These can be determined by the worker scripts, so the app has little to do here. |
```

Comment:
it might be interesting to add those information for every building footprint (e.g. for the centroid of each building), this could be generated in the backend and might help us to use the produced data also together with data from built_area projects.